pkg/stanza/fileconsumer/matcher/matcher.go

```diff
@@ -66,11 +71,21 @@ func New(c Criteria) (*Matcher, error) {
 		return nil, fmt.Errorf("exclude: %w", err)
 	}
 
+	matcher := &Matcher{
```

Comment:
Let's disambiguate from the package name
```suggestion
	m := &Matcher{
```